addon/current-user/service.js

```diff
@@ -94,6 +94,35 @@ export default Service.extend({
     this.set('_cachedUrl', null);
   },
 
+  fetchColleagues() {
+    const url = Configuration.identityURL;
+    const token = encodeURIComponent(
+      this.get('session.data.authenticated.access_token')
+    );
+
+    return this._cachedUser.then((res) => {
```

Comment:
better use `.fetch` and let it decide whether it will return a cached version or not, instead of assuming their is a `cachedUser`